build/package/Dockerfile.java-toolset

```diff
@@ -0,0 +1,41 @@
+FROM registry.access.redhat.com/ubi8
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# Add adoptopenjdk repo
+COPY build/package/yum.repos.d/adoptopenjdk.repo /etc/yum.repos.d/adoptopenjdk.repo
+
+# Install Adoptopenjdk
+RUN yum install -y wget && \
+    INSTALL_PKGS="adoptopenjdk-16-hotspot" && \
```

Comment:
Not sure if we should track LTS only?

Reply:
Yes, make sense. We don't need for this any new java feature. Let us use a LTS version, which are the ones that should be released into production.

Reply:
I put some more thoughts on this. I believe we shouldn't restrict the compilation to a specific java version. When releasing an application in production environment the general recommendation is to use a LTS version. However for compiling it, we could use a latest version, even if is not LTS. This will allow use to support java projects that are on latest version and are not meant for production release. What do you think?

Reply:
I'll keep LTS version for now